<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="inputServerAddr">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>server
address</string>
    </property>
   </widget>
   <widget class="QLabel" name="portlabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>port</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputPort">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>451</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="menuexit"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
